client/components/themes-list/style.scss

```diff
@@ -2,7 +2,7 @@
 	display: flex;
 	flex-flow: row wrap;
 	list-style: none;
-	padding: 0;
+	padding: 0 20px;
```

Comment:
@fditrapani This change has affected the themes list layout in the [theme showcase](https://wordpress.com/themes).

Should be:
<img width="185" alt="screen shot 2017-07-04 at 17 44 29" src="https://user-images.githubusercontent.com/7767559/27838140-9f458246-60e0-11e7-9983-372d57f52126.png">

is now:
<img width="193" alt="screen shot 2017-07-04 at 17 44 19" src="https://user-images.githubusercontent.com/7767559/27838145-a6063a58-60e0-11e7-83b7-f512ac01251e.png">

Seems odd that this change would be needed for a plans A/B test?